patches/server/0011-Paper-command.patch

```diff
@@ -411,7 +417,14 @@ index 0000000000000000000000000000000000000000..9e51b3d1217ad6dc5c0c11d2febac714
 +                sender.sendMessage("Entity: " + name + " Total Ticking: " + (info.getLeft() - nonTicking) + ", Total Non-Ticking: " + nonTicking);
 +                info.getRight().entrySet().stream()
 +                    .sorted((a, b) -> !a.getValue().equals(b.getValue()) ? b.getValue() - a.getValue() : a.getKey().toString().compareTo(b.getKey().toString()))
-+                    .limit(10).forEach(e -> sender.sendMessage("  " + e.getValue() + ": " + e.getKey().x + ", " + e.getKey().z + (chunkProviderServer.isPositionTicking(e.getKey().toLong()) ? " (Ticking)" : " (Non-Ticking)")));
++                    .limit(10).forEach(e -> {
++                        final int x = (e.getKey().x << 4) + 8;
++                        final int z = (e.getKey().z << 4) + 8;
++                        final Component message = text("  " + e.getValue() + ": " + e.getKey().x + ", " + e.getKey().z + (chunkProviderServer.isPositionTicking(e.getKey().toLong()) ? " (Ticking)" : " (Non-Ticking)"))
++                            .hoverEvent(HoverEvent.showText(text("Click to teleport to chunk", GREEN)))
++                            .clickEvent(ClickEvent.clickEvent(ClickEvent.Action.RUN_COMMAND, "/minecraft:execute as @s in " + world.getWorld().getKey().getKey() + " run tp " + x + " " + (world.getWorld().getHighestBlockYAt(x, z, HeightMap.MOTION_BLOCKING) + 1) + " " + z));
```

Comment:
`world.getWorld().getKey().getKey()` doesn't include the namespace. It's possible two worlds will have the same value then. This should be the fully qualified key.toString()d. So it'd be `minecraft:overworld` instead of just `overworld`.